.circleci/config.yml

```diff
@@ -449,7 +449,7 @@ workflows:
             branches:
               only:
                 - /^release-\d+(\.\d+){1,2}(-rc.*)?/
-                - pull/1695
+                - pull/1680
```

Comment:
should we remove this before merging?

Reply:
Probably. It won't affect anything after the PR is merged. I've been trying to look into a way to get signed builds made up when a label is added so that UX/QA can get test builds when they need them.